extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js

```diff
@@ -90,6 +94,9 @@ export class AmpLightboxViewer extends AMP.BaseElement {
     /** @private {!Array<!Element>} */
     this.clonedLightboxableElements_ = [];
 
+    /** @private {!Array<any>} */
```

Comment:
You should create something like a `LightboxElementMetadataDef` for the type with `descriptionText` 
 and `tagName` properties that gets stored in this array. There are a lot of good examples of these elsewhere in the core code.